### PR TITLE
Feat/circular icon buttons

### PR DESCRIPTION
**Pull request by Christos-Zacharopoulos**
## Description

In this PR we change the Icon Buttons to circular only.

[Design](https://www.figma.com/file/8kMPBNYHHXz2AtkzeeDmk5/Design-System?node-id=574%3A5912)

##  Screenshot

Capture

## Test Plan

Updated storyshots where the icon button was used.

### CODE REVIEW

**Comment by panvourtsis** [comment]
reminder

**Review by SVonKarajann — APPROVED** [review]
LGTM

File: src/components/IconButton/IconButton.style.ts [hunk 1]
```diff
@@ -1,54 +1,46 @@
-import { rem } from 'polished';
-import { Theme } from '../../theme';
+import { buttonStyle, heightBasedOnSize } from '../Button/Button.style';
```
Comment (panvourtsis) [inline src/components/IconButton/IconButton.style.ts:1]: image
something doesnt look right

**Review by panvourtsis — CHANGES_REQUESTED** [review]

File: src/components/IconButton/IconButton.style.ts (hunk 1 above)
Comment (panvourtsis) [reply src/components/IconButton/IconButton.style.ts:1]: swatches are not applied i guess for icon here as it does in the button

Comment (Christos-Zacharopoulos) [reply src/components/IconButton/IconButton.style.ts:1]: image

**Comment by Christos-Zacharopoulos** [comment]
@panvourtsis @SVonKarajann  Have added the swatches and a playground for the IconButton.

**Review by panvourtsis — APPROVED** [review]

**Comment by Christos-Zacharopoulos** [comment]
image

**Pull request merged**